Assets/HoloToolkit/Input/Scripts/Utilities/Managers/MixedRealityTeleport.cs

```diff
@@ -27,16 +27,28 @@ namespace HoloToolkit.Unity.InputModule
     public class MixedRealityTeleport : Singleton<MixedRealityTeleport>, IControllerInputHandler
     {
         [Tooltip("Name of the thumbstick axis to check for teleport and strafe.")]
-        public string LeftThumbstickX = "CONTROLLER_LEFT_STICK_HORIZONTAL";
+        public XboxControllerMappingTypes HorizontalStrafe = XboxControllerMappingTypes.XboxLeftStickHorizontal;
 
         [Tooltip("Name of the thumbstick axis to check for teleport and strafe.")]
-        public string LeftThumbstickY = "CONTROLLER_LEFT_STICK_VERTICAL";
+        public XboxControllerMappingTypes VerticalStrafe = XboxControllerMappingTypes.XboxLeftStickVertical;
 
         [Tooltip("Name of the thumbstick axis to check for rotation.")]
-        public string RightThumbstickX = "CONTROLLER_RIGHT_STICK_HORIZONTAL";
+        public XboxControllerMappingTypes HorizontalRotation = XboxControllerMappingTypes.XboxRightStickHorizontal;
 
         [Tooltip("Name of the thumbstick axis to check for rotation.")]
-        public string RightThumbstickY = "CONTROLLER_RIGHT_STICK_VERTICAL";
+        public XboxControllerMappingTypes VerticalRotation = XboxControllerMappingTypes.XboxRightStickVertical;
+
+        [Tooltip("Custom Input Mapping for horizontal teleport and strafe")]
+        public string LeftThumbstickX = InputMappingAxisUtility.CONTROLLER_LEFT_STICK_HORIZONTAL;
+
+        [Tooltip("Custom Input Mapping for vertical teleport and strafe")]
+        public string LeftThumbstickY = InputMappingAxisUtility.CONTROLLER_LEFT_STICK_VERTICAL;
+
+        [Tooltip("Custom Input Mapping for horizontal rotation")]
+        public string RightThumbstickX = InputMappingAxisUtility.CONTROLLER_RIGHT_STICK_VERTICAL;
+
+        [Tooltip("Custom Input Mapping for vertical rotation")]
+        public string RightThumbstickY = InputMappingAxisUtility.CONTROLLER_RIGHT_STICK_HORIZONTAL;
```

Comment:
This should be `InputMappingAxisUtility.CONTROLLER_RIGHT_STICK_VERTICAL`

Reply:
So it's backwards? I used the mapping from prev version.

Reply:
Yeah, it's backwards from the previous version.

![image](https://user-images.githubusercontent.com/3580640/33690836-438ee6fc-da9a-11e7-95e1-69c84f01f171.png)